docs/4.4.yaml

```diff
@@ -33,7 +33,7 @@ extra_javascript: []
 extra:
     version: 4.4
     teleport:
-        version: 4.4.1
+        version: 4.4.2
         golang: 1.14
         sha: 77eb69c1ad4a536ecbb02a8c842f53576831ae8c95c1295091b0b5d90fb2643f
```

Comment:
Think this will need updating right?

Reply:
We actually don't use the SHA example anymore in 4.4 examples, it's less likely we'll need it but I'm still a little attached to it :-)